src/contents/shared/ProfileEdit.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { Component } from 'react';
+import fire from '../../config/Fire';
+import { ProfileEditView } from './ProfileEditView';
+import firebase from 'firebase';
+import { storage } from '../../config/Fire';
+
+export class ProfileEdit extends Component {
+	constructor(props) {
```

Comment:
please look at how the rest of our code looks. We never use constructor instead you can write directly in the class:
`state = { ... }`

Reply:
Done!